### ISSUE TITLE
Many of our "learn more" links are unhelpful

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#30085]
We seem to override all support.google.com links in https://github.com/brave/brave-core/blob/master/chromium_src/chrome/common/url_constants.cc and for many of them, we simply link to the https://support.brave.com/ since we don't have documentation for that setting/feature.

I realize that it may be surprising to users to be taken to a Chrome support page, but our generic links are not helpful to users.

One option would be to remove the links altogether. Another would be to figure out what documentation we are missing and actually write it. In some cases, it may be sufficient to say that more details can be found at https://support.google.com/whatever-specific-page-upstream-links-to.

**Comment by fmarier** [comment]
For a concrete example (to be fixed in https://github.com/brave/brave-browser/issues/20514), go into `brave://settings/security` and then click the question mark icon in the top-right corner.

**Comment by rmcfadden3** [comment]
> For a concrete example (to be fixed in #20514), go into `brave://settings/security` and then click the question mark icon in the top-right corner.

Yeah, I tend to agree. In this instance the `?` is confusing. At the very least, we should add hover text on the `?` that says like `Visit Brave help center for help with settings`

But an even better solution might be to remove the `?` altogether, and instead have a string at the bottom of the page that says like `Need help with security settings? Check out the Brave [help center](https://support.brave.com/) or [community forums](https://community.brave.com/).`
